src/enums/order.info.status.ts

```diff
@@ -1,6 +1,7 @@
 export enum OrderInfoStatus {
```

Comment:
This enum is for internal structures. Please give proper `PascalCase` keys and remove values (they are not tied to any external structure)